JASP-Desktop/po/jasp_nl.po

```diff
@@ -16078,10 +15845,2124 @@ msgctxt "RibbonModel|"
+
+msgctxt "MixedModelsPlots|"
+msgid "Confidence interval"
+msgstr "Betrouwbaarheidsinterval"
+
+msgctxt "MixedModelsPlots|"
+msgid "Distinguish factor levels"
+msgstr "Onderscheid factorniveau's"
+
+msgctxt "MixedModelsPlots|"
+msgid "Color"
+msgstr "Kleur"
+
+msgctxt "MixedModelsPlots|"
+msgid "Shape"
+msgstr "Vorm"
+
+msgctxt "MixedModelsPlots|"
+msgid "Linetype"
+msgstr "Lijntype"
+
+msgctxt "MixedModelsPlots|"
+msgid "Fill"
+msgstr "Vul"
+
+msgctxt "MixedModelsPlots|"
+msgid "Background geom"
+msgstr "Achtergrond geom"
+
+msgctxt "MixedModelsPlots|"
+msgid "Jitter"
+msgstr "Jitter"
+
+msgctxt "MixedModelsPlots|"
+msgid "Violin"
+msgstr "Viool"
+
+msgctxt "MixedModelsPlots|"
+msgid "Boxplot"
+msgstr "Boxplot"
+
+msgctxt "MixedModelsPlots|"
+msgid "Boxjitter"
+msgstr "Boxjitter"
+
+msgctxt "MixedModelsPlots|"
+msgid "Count"
+msgstr "Aantal"
+
+msgctxt "MixedModelsPlots|"
+msgid "Transparency"
+msgstr "Transparantie"
+
+msgctxt "MixedModelsPlots|"
+msgid "Jitter width"
+msgstr "Jitter breedte"
+
+msgctxt "MixedModelsPlots|"
+msgid "Jitter height"
+msgstr "Jitter hoogte"
+
+msgctxt "MixedModelsPlots|"
+msgid "Geom width"
+msgstr "Geom breedte"
+
+msgctxt "MixedModelsPlots|"
+msgid "Dodge"
+msgstr "Ontwijken"
+
+msgctxt "MixedModelsPlots|"
+msgid "Theme"
+msgstr "Thema"
+
+msgctxt "MixedModelsPlots|"
+msgid "Black White"
+msgstr "Zwart Wit"
+
+msgctxt "MixedModelsPlots|"
+msgid "Light"
+msgstr "Licht"
+
+msgctxt "MixedModelsPlots|"
+msgid "Minimal"
+msgstr "Minimaal"
+
+msgctxt "MixedModelsPlots|"
+msgid "Legend position"
+msgstr "Legenda positie"
+
+msgctxt "MixedModelsPlots|"
+msgid "Bottom"
+msgstr "Beneden"
+
+msgctxt "MixedModelsPlots|"
+msgid "Right"
+msgstr "Rechts"
+
+msgctxt "MixedModelsPlots|"
+msgid "Top"
+msgstr "Boven"
+
+msgctxt "MixedModelsPlots|"
+msgid "Left"
+msgstr "Links"
+
+msgctxt "MixedModelsPlots|"
+msgid "Color background data"
+msgstr "Kleur achtergrond gegevens"
+
+msgctxt "MixedModelsPlots|"
+msgid "Dark grey"
+msgstr "Donkergrijs"
+
+msgctxt "MixedModelsPlots|"
+msgid "Black"
+msgstr "Zwart"
+
+msgctxt "MixedModelsPlots|"
+msgid "Light grey"
+msgstr "Lichtgrijs"
+
+msgctxt "MixedModelsPlots|"
+msgid "Blue"
+msgstr "Blauw"
+
+msgctxt "MixedModelsPlots|"
+msgid "Red"
+msgstr "Rood"
+
+msgctxt "MixedModelsPlots|"
+msgid "Violet"
+msgstr "Paars"
+
+msgctxt "MixedModelsPlots|"
+msgid "Relative size text"
+msgstr "Relatieve tekstgrootte"
+
+msgctxt "MixedModelsPlots|"
+msgid "Relative size foreground data"
+msgstr "Relatieve voorgrond gegevensgrootte"
+
+msgctxt "MixedModelsPlots|"
+msgid "Estimates table"
+msgstr "Schattingstabel"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Binomial"
+msgstr "Binomiaal"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Binomial (aggregated)"
+msgstr "Binomiaal (geaggregeerd)"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Gaussian"
+msgstr "Gaussische"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Gamma"
+msgstr "Gamma"
+
+msgctxt "MixedModelsBGLMM|"
+msgid "Inverse Gaussian"
+msgstr "Inverse Gaussiaans"
```

Comment:
Gaussiaans of Gaussische? Zie 2 vertalingen hierboven

Reply:
Gaussiaan!

Reply:
Hah, geen idee ook.